src/mixs/schema/ancient.yml

```diff
@@ -367,26 +368,27 @@ slots:
       partial_match: true
   sample_age:
     annotations:
+      Expected_value: value or range starting from from BP (1950)
       Preferred_unit: in years before present (BP) or (BCE/CE)
     description: >-
-      The approximate date that individual was living and then died, or the 
+      The approximate date or date range that individual was living and then died, or the 
```

Comment:
Just wondering of individual is right here since we could also have plants etc.

Reply:
This is now required but there is no guidance in case the data is not available.